openmc/plots.py

```diff
@@ -401,14 +401,13 @@ def get_plot_xml(self):
         return element
 
 
-class PlotsFile(object):
+class Plots(object):
     """Plots file used for an OpenMC simulation. Corresponds directly to the
```

Comment:
Perhaps "Collection of Plots used for an OpenMC simulation" would be an alternative wording of the docstring to better conform to the new class name?